### DISCUSSION TITLE
How to split Default.less into 2 separate files

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#200] [1. Help]
Hi, 
I want to split Default.less file into 2 separate files i.e default-one.less and default-second.less
How can I do that?
I am unable to find from where we are including default.less file.
Can you please guide?

**Comment by vovayatsyuk** [comment]
How to create a custom css bundle: https://breezefront.com/docs/custom-styles#custom-css-bundle

I don't think you should edit the default css bundle as you will not be able to update your theme in the future. However, if you really want to do this: https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/web/css/default.less.
